api/v1/attesterduty_test.go

```diff
@@ -18,8 +18,8 @@ import (
 	"testing"
 
 	api "github.com/attestantio/go-eth2-client/api/v1"
-	require "github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/assert"
+	require "github.com/stretchr/testify/require"
```

Comment:
There seem to be a number of items in here that shouldn't be part of the PR; please can you remove them?

Reply:
Ah yes sorry I'm not sure how those made it in via the lint.